vllm/README.md

```diff
@@ -22,3 +22,28 @@ Once the server is deployed, you can use the `yichat.py` script to chat with the
 ```bash
 python yichat.py
 ```
+
+## Multi-modal inference
+
+We can deploy a multi-modal model like Phi-3.5-vision-instruct. The `phi.py` script demonstrates how to do this.
+
+```bash
+beam deploy phi.py:phi
```

Comment:
this command is confusing -- i'd rename phi.py to inference.py or something like that